### PR TITLE
New package: sbs-1.0.0

### PR DESCRIPTION
**Pull request by ozkanonur**
#### Testing the changes
* I tested the changes in this PR: **YES**
 
#### New package
* This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**
 
[feh](https://github.com/derf/feh) alternative simple background setter. Useful for WM users.

### CODE REVIEW

File: srcpkgs/sbs/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'sbs'
+pkgname=sbs
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libXinerama-devel"
+depends="imlib2-devel libXinerama-devel"
```
Comment (classabbyamp) [inline srcpkgs/sbs/template:9]: unnecessary, should be auto-detected

File: srcpkgs/sbs/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+maintainer="Onur Ozkan <work@onurozkan.dev>"
+license="GPL-2.0-only"
+homepage="https://github.com/ozkanonur/sbs"
+changelog="https://raw.githubusercontent.com/ozkanonur/sbs/master/ChangeLog"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=aa4da5b2315046fc2059599b19c530f08bb870e63ed17111a55991b1ae911367
+
+do_install() {
+	vbin sbs
+}
```
Comment (classabbyamp) [inline srcpkgs/sbs/template:20]: why not let the default do_install do it? it should run `make install` just fine

File: srcpkgs/sbs/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libXinerama-devel"
+depends="imlib2-devel libXinerama-devel"
+short_desc="single purpose non-bloat background setter"
+maintainer="Onur Ozkan <work@onurozkan.dev>"
+license="GPL-2.0-only"
+homepage="https://github.com/ozkanonur/sbs"
+changelog="https://raw.githubusercontent.com/ozkanonur/sbs/master/ChangeLog"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/sbs/template:15]: don't use homepage here

**Review by classabbyamp — COMMENTED** [review]
[fix the commit message too](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#committing-your-changes)

File: srcpkgs/sbs/template (hunk 1 above)
Comment (ozkanonur) [reply srcpkgs/sbs/template:9]: Compilation fails on `xbps-src pkg sbs` when I exclude those. Am I missing something?

File: srcpkgs/sbs/template (hunk 3 above)
Comment (ozkanonur) [reply srcpkgs/sbs/template:15]: Sorry, I couldn't get what's wrong with it. Shouldn't it be the repository link?

File: srcpkgs/sbs/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/sbs/template:9]: remove them only from depends, not makedepends

File: srcpkgs/sbs/template (hunk 3 above)
Comment (classabbyamp) [reply srcpkgs/sbs/template:15]: expand the variable to its value

Comment (ozkanonur) [reply srcpkgs/sbs/template:15]: Ah, got you now I was on completely different point xd

**Pull request closed without merging**